### ISSUE TITLE
Formatting improvement

### ISSUE THREAD

**Issue by Trefex** [pytr-org/pytr#96]
For portfolio, the output could not look nice because of long asset name

```
21:52:31 Connected to websocket ...
Name                      ISIN            avgCost *   quantity =    buyCost ->   netValue       diff   %-diff
PepsiCo                   US7134481081     162.66 *       7.38 =    1200.00 ->    1122.66     -77.34    -6.4%
Core STOXX Europe 600 EUR (Acc) LU0908500753     205.87 *       7.04 =    1450.00 ->    1704.79     254.79    17.6%
Coca-Cola                 US1912161007      55.73 *      30.74 =    1712.76 ->    1790.65      77.89     4.5%```

**Comment by Katzmann1983** [comment]
I thought I was the only one annoyed by that. It's a less than one liner to change :)

**Comment by Katzmann1983** [comment]
Fixed with PR #101
